modules/light-clients/09-localhost/client_state.go

```diff
@@ -66,20 +66,22 @@ func (cs ClientState) Initialize(ctx sdk.Context, cdc codec.BinaryCodec, clientS
 	return nil
 }
 
-// GetTimestampAtHeight must return the timestamp for the consensus state associated with the provided height.
+// GetTimestampAtHeight returns the current block time retrieved from the application context. The localhost client does not store consensus states and thus
+// cannot provide a timestamp for the provided height.
 func (cs ClientState) GetTimestampAtHeight(ctx sdk.Context, clientStore sdk.KVStore, cdc codec.BinaryCodec, height exported.Height) (uint64, error) {
```

Comment:
should we underscore `clientStore`. `cdc` and `height`?

Reply:
Sure, now discarding all unused args across the file!